man/tuned-profiles-atomic.7

```diff
@@ -59,6 +59,7 @@ netfilter connections tracking.
 .BR tuned\-profiles\-nfv\-guest (7)
 .BR tuned\-profiles\-cpu\-partitioning (7)
 .BR tuned\-profiles\-compat (7)
+.BR tuned\-profiles\-postgresql (7)
```

Comment:
@yarda Do we want to keep maintaining this list in each profile man page? How about doing the following instead:
1. from each profile man page, refer to tuned-profiles(7)
2. from tuned-profiles(7), refer to every profile man page

Reply:
Yes, it makes sense. It will help maintainability. Maybe, to stay consistent I wouldn't change it here, but later by some commit changing it globally for all profiles.

Reply:
I agree with @yarda. Such a change shouldn't be the part of this PR.